net/v2ray-core/Makefile

```diff
@@ -30,9 +30,10 @@ include $(INCLUDE_DIR)/package.mk
 include ../../lang/golang/golang-package.mk
 
 define Package/v2ray/template
-  TITLE:=A platform for building proxies to bypass network restrictions
+  TITLE:=Proxy platform to bypass censorship
   SECTION:=net
   CATEGORY:=Network
+  SUBMENU:=Web Servers/Proxies
```

Comment:
I would say it's not just a proxy server/client, but a powerful networking tool. It can be used for other purpose like DNS server, port forwarder etc. Putting it to Network is better IMO.

Reply:
It will cause subentries to be hidden
![Screenshot_2024-11-17_040343](https://github.com/user-attachments/assets/ff81200e-0da6-4cd2-9174-a54dea9decce)
![Screenshot_2024-11-17_042356](https://github.com/user-attachments/assets/3d4f6264-9812-4add-87d7-b858b02e0e9f)

Reply:
Are we going to do this?

Reply:
> It will cause subentries to be hidden

I think it's good enough, as the title below is just appending package name to the end.

Reply:
> I would say it's not just a proxy server/client, but a powerful networking tool. It can be used for other purpose like DNS server, port forwarder etc. Putting it to Network is better IMO.

Okay I understand this, in most cases people just use it as an anti-censorship tool... Dropping this for both xray and v2ray